clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/mark_node_failed.py

```diff
@@ -28,20 +24,23 @@ def make_key(site, node_type, datatore):
 dead_node_ip = sys.argv[5]
 
 key = make_key(site, node_type, datastore)
-print "Using etcd key %s" % (key)
+logging.info("Using etcd key %s" % (key))
 
 error_syncer = EtcdSynchronizer(NullPlugin(key), dead_node_ip, etcd_ip=local_ip, force_leave=True)
 
+print "Marking node as failed to allow current operation to complete..."
```

Comment:
This is misleading as this isn't necessarily being run during a scaling operation